Autosniper rework/rebalance

Pariah919 opened the pull request:
<!-- ***STOP!***  Read this: If this is not a PR ready for review and merge or WIP, open it as a draft PR, using the arrow next to 'Create Pull Request'>

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Autosniper get's a fat rebalance to make it stand out more in the game.

--
Ammo stats
80D/30P/7.5S/2 min range/0.25 falloff-> 50D/30P/5S
Changes are required with the much higher rate of fire of the new Autosniper, might need to drop pen or sunder further. Min range isn't required anymore.
--
Weapon stats
Handling unchanged, attachments unchanged for now.
Magazine- 12->20
The autosniper has frankly, terrible ammo economy, this should help with it's new place.
Fire delay
1.75->0.4
This makes the autosniper frankly stand out compared to the crowd of other power weapons, and makes it more of an autosniper that can perform in any range, but has hefty wield and aim slowdown, allows it to perform decently in a supporting role rather than as a very overpriced TL with a fancy NVG that sometimes works.
--
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The autosnpier has several main issues, it struggles to stand out in any form compared to it's peers, it's not cheap, it's ammo eco sucks, the gun has no real feel.
This aims to give it some level of a niche that feels good for the gun.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
balance: Autosniper rework, bullet nerfed down to 50 damage, 5 sunder. Fire delay buffed form 1.75 seconds to 0.4, magazine buffed from 12 to 20. Handling unchanged, this aims to make the Autosniper perform better at any range and give it more of a niche as an autosniper. Price unchanged.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Pariah919 commented:
actually these stats are wack i'll redo them when i wake up tommorrow

RenGusta commented:
Damage per mag in the same park as the T12 is sensible for an autosniper. 1000 vs 1250 for the T12. It's not unreasonable. Maybe like .6 fire delay instead of .4

David-Stormwell commented:
> actually these stats are wack i'll redo them when i wake up tommorrow

do so when

Pariah919 commented:
changed sunder to 2, changed firerate to 0.5.
This should make it more balanced, and generally keeps it as a marksman weapon that can touch at close range. I'll try it like this.

David-Stormwell commented:
sure ok, but it should be renamed to a smart dmr or something.
